tests/assert-jobs/index/main.go

```diff
@@ -111,37 +113,45 @@ func initCmd() error {
 func main() {
 	err := initCmd()
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		logrus.Fatalln(err)
 	}
 
-	log = *logger.InitLog(viper.GetBool(flagVerbose))
+	if viper.GetBool(flagVerbose) == true {
+		logrus.SetLevel(logrus.DebugLevel)
+	}
+	logrus.SetOutput(os.Stdout)
```

Comment:
doesn't logrus use `os.Stdout` by default? And if not, would it make sense to place this call on top of the main function?

Reply:
That's true! Thanks